web/src/topic_popover.js

```diff
@@ -157,13 +157,22 @@ export function initialize() {
                     stream_popover.build_move_topic_to_stream_popover(stream_id, topic_name, true);
                     popover_menus.hide_current_popover_if_visible(instance);
                 });
-
-                new ClipboardJS($popper.find(".sidebar-popover-copy-link-to-topic")[0]).on(
-                    "success",
-                    () => {
+                // copy link to the stream topic
+                $popper.on("click", ".sidebar-popover-copy-link-to-topic", () => {
+                    const $topicLinkElement = $(".sidebar-popover-copy-link-to-topic");
+                    const formatedLinkText = `
+                    <a href=${$topicLinkElement.data("clipboard-text")}>${try_stream_topic_syntax_text($topicLinkElement.data("clipboard-text")).replaceAll(/\**/g, "\u200B")}</a>
+                   `;
```

Comment:
Please read https://zulip.readthedocs.io/en/stable/subsystems/html-css.html and apply it to your code; we don't use camelCase, etc.